### PR TITLE
Sonar fix master

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/summary/utils.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const getSectionSummaryList = (
 
 const setkey = (userCase: Partial<CaseWithId>, key: string, language: string | undefined) => {
   const userkey = userCase[key];
+  let welshtranslation;
```
Comment (vivek-sekhar) [inline src/main/steps/common/summary/utils.ts:50]: don't use specifics as variable names.

File: src/main/steps/common/summary/utils.ts [hunk 2]
```diff
@@ -47,51 +47,58 @@ export const getSectionSummaryList = (
...
-  if (key === 'courtProceedingsOrders' && !userCase[key]) {
-    return getOrdersDetail(userCase);
-  }
-  if (key === 'citizenUserAddressHistory' && userCase['isAtAddressLessThan5Years'] === YesOrNo.YES) {
-    return userCase['citizenUserAddressText'];
-  }
+  // if (key === 'citizenUserAddressHistory' && userCase['isAtAddressLessThan5Years'] === YesOrNo.YES) {
+  //   return userCase['citizenUserAddressText'];
+  // }
+  let yesOrNoWelshLabel;
```
Comment (vivek-sekhar) [inline src/main/steps/common/summary/utils.ts:53]: Rename it translationLabel

File: src/main/steps/common/summary/utils.ts [hunk 3]
```diff
@@ -47,51 +47,58 @@ export const getSectionSummaryList = (
...
-      welshtranslation = 'doTranslation';
+      yesOrNoWelshLabel = 'doTranslation';
       break;
     case 'miamWillingness':
-      welshtranslation = 'byddwnTranslation';
+      yesOrNoWelshLabel = 'byddwnTranslation';
       break;
+    case 'courtProceedingsOrders':
+      if (!userCase[key]) {
+        return getOrdersDetail(userCase);
```
Comment (vivek-sekhar) [inline src/main/steps/common/summary/utils.ts:82]: why returns are used for few case statements alone? make it consistent and return at the end of the function.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/summary/utils.ts (hunk 2 above)
Comment (SasankAchyuta9) [reply src/main/steps/common/summary/utils.ts]: Done

**Review by vivek-sekhar — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**